.stickler.yml

```diff
@@ -0,0 +1,21 @@
+
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+    
+  sasslint:
+    # indicate where is the config file for stylelint
+    config: ./sass-lint.yml
```

Comment:
Concerning your sass lint issues:
This file does not exist. it should be your `.stylelintrc.json` not so?
Maybe this should help. Once your config is right it should be fine.